selftests/functional/test_skiptests.py

```diff
@@ -32,15 +33,16 @@ def test2(self):
     def test3(self):
         pass
 
-    @avocado.skipIf(False)
+    @avocado.skipIf(False, "won't be skipped")
     def test4(self):
         pass
 
-    @avocado.skipUnless(True)
+    @avocado.skipUnless(True, "won't be skipped")
```

Comment:
Oups, my fault. Sorry I did this in a hurry just to demonstrate this version... I'll wait for the response and then I'll polish it (if accepted)